app/lib/backLinkUtils.js

```diff
@@ -1,28 +1,13 @@
-const contexts = require('./contexts');
-
-function text(res) {
-  return res.locals.context === contexts.stomachAche.context
-    ? contexts.stomachAche.text
-    : 'Back';
-}
-
-function url(req, res) {
-  let link;
-
-  if (res.locals.context === contexts.stomachAche.context) {
-    link = contexts.stomachAche.url;
-  } else {
-    // eslint-disable-next-line no-script-url
-    link = req.get('referer') || 'javascript:history.back();';
-  }
-
+function url(req) {
+  // eslint-disable-next-line no-script-url
+  const link = req.get('referer') || 'javascript:history.back();';
   return link;
 }
 
-function backLinkUtils(req, res) {
+function backLinkUtils(req) {
   return {
-    text: text(res),
-    url: url(req, res),
+    text: 'Back',
+    url: url(req),
```

Comment:
This line could be `url: req.get('referer') || 'javascript:history.back();',` and then the function `url` could be removed. Saves some lines, maybe makes it easier to read?